client/src/components/navbar/NavBar.css

```diff
@@ -0,0 +1,30 @@
+.top-navbar {
+    width: 100%;
+    height: 11%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0;
+    margin: 0;
+    border-bottom: 3px solid #CECECE;
```

Comment:
In the figma we have the border only extending partially across the width rather than just the bottom. No worries if this is an intentional change to make it span the whole width! Just double checking